ui/components/app/cancel-speedup-popover/index.scss

```diff
@@ -0,0 +1,27 @@
+.cancel-speedup-popover {
+  &__wrapper {
+    padding: 0 16px 16px;
+
+    .info-tooltip {
+      margin-left: 4px;
+    }
+  }
+
+  .edit-gas-fee-button {
```

Comment:
This adds specificity to this selector which may not be desirable if we plan to reuse this component for something else in the future. Is it possible to pass a `className` to EditGasFeeButton, similar to what you did for AppLoadingSpinner? That way this could be `&__edit-gas-fee-button` instead.

Reply:
fixed it